core/src/main/java/io/questdb/griffin/model/CreateTableModel.java

```diff
@@ -183,6 +185,10 @@ public int getTimestampIndex() {
         return timestamp == null ? -1 : getColumnIndex(timestamp.token);
     }
 
+    public CharSequence getVolumePath() {
```

Comment:
Not used, except to compare with null, looks like not ready.

Reply:
as above

Reply:
not as above, path is not used, please explain how it works give the test name where it's tested